.github/workflows/image-publish-check-on-push-scheduled.yml

```diff
@@ -39,4 +40,92 @@ jobs:
     with:
       image_ref: 'ghcr.io/${{ github.repository_owner }}/dbildungs-iam:latest'
       fail_on_vulnerabilites: false
-      report_location: "Dockerfile"
\ No newline at end of file
+      report_location: "Dockerfile"
+
+  scan_helm:
+    if: ${{ github.event_name == 'push' && !startsWith(github.ref_name,'dependabot/') }}
+    uses: dBildungsplattform/dbp-github-workflows/.github/workflows/check-helm-kics.yaml@2.1.0
+    permissions:
+      contents: read
+
+  select_helm_tag_and_version_generation:
+    if: ${{ github.event_name == 'push'}}
+    needs: 
+      - scan_helm
+    runs-on: ubuntu-latest
+    outputs: 
+      SELECT_HELM_VERION_GENERATION: ${{ steps.select_generation.outputs.SELECT_HELM_VERION_GENERATION }}
+      SELECT_IMAGE_TAG_GENERATION: ${{ steps.select_generation.outputs.SELECT_IMAGE_TAG_GENERATION }}
+    steps:
+      - id: select_generation
+        shell: bash
+        run: |
+          if ${{ github.ref_name == 'main' }}; then
+              echo "SELECT_HELM_VERION_GENERATION=short_hash" >> "$GITHUB_OUTPUT"
+              echo "SELECT_IMAGE_TAG_GENERATION=commit_hash" >> "$GITHUB_OUTPUT"
+          else
+              echo "SELECT_HELM_VERION_GENERATION=ticket_from_branch_short_hash" >> "$GITHUB_OUTPUT"
+              echo "SELECT_IMAGE_TAG_GENERATION=ticket_from_branch" >> "$GITHUB_OUTPUT"
+          fi
+  release_helm:
+    needs:
+      - select_helm_version_generation
```

Comment:
This should be changed to select_helm_tag_and_version_generation (job renamed). Otherwise the workflow fails (https://github.com/dBildungsplattform/dbildungs-iam-keycloak/actions/runs/6980924782).